donut/static/js/groups/positions.js

```diff
@@ -151,6 +152,51 @@ function groupPosHoldChange() {
     populateListOfPositions(groupIndex, '#posIdHold');
 }
 
+/*
+ * Called when a group is selected for the "delete position holder" tab.
+ * Queries for all positions associated with the group
+ */
+function groupDelPosHoldChange() {
+    var groupIndex = $('#groupDelPosHold').val();
+    $('#posIdDelHold').find('option').remove();
+    $('#posIdDelHold').append('<option> Pick a position </option>');
+    populateListOfPositions(groupIndex, '#posIdDelHold');
+}
+
+/*
+ * Converts a dateString to a more readable format
+ */
+function formatDate(dateString) {
+    if (dateString == null) {
```

Comment:
Don't think the `dateString == null` case should be handled inside this function. Would make more sense for the caller to deal with this. Imagine what happens when `data[i].start_date` and `data[i].end_date` are `null`: you will see something like `(null to null)` as the option value. Are these dates even allowed to be `NULL` in the database?

Reply:
The issue is that in the permission holders table, null date values are allowed. 

In the future when the Current_Position_holders View comes online we won't run into this issue. Since we're still pulling from data that could have null start/end date (and we still might want to see those positions), i think it makes sense to have it here for now.